operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -199,6 +257,21 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                 LOG.finest("The request specified a '" + START_QUERY_PARAMETER + "' and/or '" + END_QUERY_PARAMETER + "' query parameter. They are not valid for resource type '" + compartmentType + "', so will be ignored.");
                 searchParameters = queryParametersWithoutDates;
             }
+            
+            // Need to get the value of the _type parameter out, if it exists
+            List<String> typeParameters = new ArrayList<String>(0);
+            List<Parameters.Parameter> parameterList = parameters.getParameter();
+            for (Parameters.Parameter parameter: parameterList) {
+                typeParameters = new ArrayList<String>(parameterList.size());
+                if (parameter.getName().getValue().equals("_type")) {
+                    String[] types = parameter.getValue().as(ModelSupport.FHIR_STRING).getValue().split(",");
```

Comment:
I think our operation framework will guarantee this has the right type, but if not it would be a classcastexception.

If we don't have one yet, it would be good to add a test that confirms this gets returned to the client as a client error (400) and not a server error (500).

Reply:
Since this code was removed, I will not address this comment.